cmd/subnetcmd/upgradecmd/export_upgrade.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package upgradecmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ava-labs/avalanche-cli/pkg/constants"
+	"github.com/ava-labs/avalanche-cli/pkg/ux"
+	"github.com/ava-labs/avalanchego/utils/storage"
+	"github.com/spf13/cobra"
+)
+
+// avalanche subnet upgrade import
+func newUpgradeExportCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export [subnetName]",
+		Short: "Export the upgrade bytes file to a location of choice on disk",
+		Long:  `Export the upgrade bytes file to a location of choice on disk`,
+		RunE:  upgradeExportCmd,
+		Args:  cobra.ExactArgs(1),
+	}
+
+	cmd.Flags().StringVar(&upgradeBytesFilePath, upgradeBytesFilePathKey, "", "Export upgrade bytes file to location of choice on disk")
+	if err := cmd.MarkFlagRequired(upgradeBytesFilePathKey); err != nil {
```

Comment:
Let's not use required flags. Prompt for the location instead if it's not provided.